Adding Vestigo URL Router

husobee opened the pull request:
Adding Vestigo - implements a performant, stand-alone, HTTP compliant URL Router for go web applications.

matrixik commented:
Hello, thank you for contribution and sorry it took so long to check it.

Before merging can you fix `go vet` errors: https://goreportcard.com/report/github.com/husobee/vestigo
And could you rebase README against master?

husobee commented:
No problem.  I have updated vestigo based on the suggestions from `go vet` and I have rebased README.md.  Let me know if there is anything else I can do.

matrixik commented:
Thank you.
